hack/windows/Prepare-Node.ps1

```diff
@@ -1,37 +1,30 @@
 <#
 .SYNOPSIS
-(Test Only)Assists with preparing a Windows VM prior to calling kubeadm join
+Assists with preparing a Windows VM prior to calling kubeadm join
 
 .DESCRIPTION
-This script is only used for test to assist with joining a Windows node to a cluster.
-For production environment please follow the antrea windows installation guide and use Kubernetes official PrepareNode.ps1
-script from the sig-windows-tools repo release page: https://github.com/kubernetes-sigs/sig-windows-tools/releases
+This script assists with joining a Windows node to a cluster.
 - Downloads Kubernetes binaries (kubelet, kubeadm) at the version specified
-- Registers wins as a service in order to run kube-proxy and antrea-agent as DaemonSets.
+- Registers wins as a service in order to run kube-proxy and cni as DaemonSets.
 - Registers kubelet as an nssm service. More info on nssm: https://nssm.cc/
-- Adds node ip in kubelet startup params.
-- Create virtual netadapter for kube-proxy.
-- (Optional) Installs OVS kerner driver, userspace binaries. Registers ovsdb-server and ovs-vswitchd services.
 
 .PARAMETER KubernetesVersion
 Kubernetes version to download and use
 
-.PARAMETER InstallOVS
-Install OVS
-
-.PARAMETER NodeIP
-The node ip used by kubelet
+.PARAMETER ContainerRuntime
+Container that Kubernetes will use. (Docker or containerD)
```

Comment:
hard for people to remember this writings, "containerD" is not well known.
I think just docker or containerd.